Vagrantfile

```diff
@@ -1,6 +1,22 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+require 'yaml'
+
+config_file = 'config.yaml'
+config = YAML.load_file("#{config_file}")
```

Comment:
To avoid making changes in config file which is under git, it is better to give option to create own config by copying original to 'local.config' directory (is currently used for file with github token), which is in gitignore.
Just merge this file (if exists) over the original config file.